CDP4ShellDialogsTestFixture/HomeRibbonTestFixture.cs

```diff
@@ -4,7 +4,7 @@
 // </copyright>
 // --------------------------------------------------------------------------------------------------------------------
 
-namespace CDP4EngineeringModel.Tests.RibbonControls
+namespace CDP4ShellDialogs.Tests.RibbonControls
```

Comment:
It would seem that this file is no longer needed, does not have any tests that I see.

Reply:
Test case has been removed